### ISSUE TITLE
Agenda for Aug 31st, 2023

### ISSUE THREAD

**Issue by nairnandu** [web-platform-tests/interop#400]
Here is the proposed agenda for the Aug 31st meeting:
- Review and finalize [proposed changes](https://github.com/web-platform-tests/interop/pull/102#discussion_r1304706740) to the charter PR #102 
- Review and unblock landing of #394 
- Review and finalize Interop 2024 [schedule](https://github.com/web-platform-tests/interop/blob/apple-interop-2024-proposal-process/2024/planning-process.md#timeline--10-weeks-total)
- Discuss voting procedures for the 2024 focus areas #390 
- Communications plan - call for proposals
- Review and take action on [test change proposals](https://github.com/web-platform-tests/interop/issues?q=is%3Aissue+is%3Aopen+label%3Atest-change-proposal)

**Comment by nairnandu** [comment]
Attendees: @nairnandu, @jensimmons, @boazsender, @nt1m, @chrishtr, @zcorpan, @meyerweb, @tantek, @slightlyoff, @gsnedders, @bkardell

Notes
*   Review and finalize [proposed changes](https://github.com/web-platform-tests/interop/pull/102#discussion_r1304706740) to the charter PR [Create the Interop Team Charter #102](https://github.com/web-platform-tests/interop/pull/102)
    *   meyerweb: agree with the proposed changes
    *   Google and Bocoup in favor of the proposed change via “likes” on the comment
    *   chrishtr: Google agrees on the changes proposed by dandclark
    *   jensimmons: if we don’t have consensus to make anything confidential, it will be public. I don't believe Apple is going to agree to that. There are a multitude of considerations at play and it is easily misunderstood. This is more of a roadmap conversation.
    *   boazsender: what if its public by default, but if someone wants to keep something confidential, they can vote to do so (not requiring consensus to make decisions confidential)
    *   slightlyoff: our sticking point is the idea that there are unlimited, silent vetoes that we can't even discuss the existence of
    *   tantek: the point about group cohesion resonates. Fine to leave the charter verbiage as-is. 
    *   slightlyoff: the legitimacy of this effort hinges on the idea that interop agitates for progress on the most basic, most reasonable needs of developers. Having that undermined by the existence of private vetos is rough. This will out \_somehow\_, and it will be bad for the group's cohesion when it does.
    *   bkardell: this is an opportunity to align our roadmaps.
    *   boazsender: agree on the cohesion and being able to talk openly in this group as a priority
    *   jensimmons: charter is not meant to debate individual issues and maybe we can separate that out as a different issue.
    *   tantek: in favor of writing down things that have worked well for us as a group
    *   chrishtr: next step would be to withdraw the comment on the current PR and create another PR for the proposed changes
    *   tantek: suggest that an issue be created with specific reasoning, as opposed to a PR
    *   slightlyoff: not in favor of carving this out as a separate PR/issue.
    *   Next step: since we do not have consensus, move this topic to next week’s agenda 
*   Review and unblock landing of [Add information about the Interop Project to the README #394](https://github.com/web-platform-tests/interop/pull/394)
    *   nanrinandu: General consensus on landing this PR, once the comments are resolved
    *   Next step: gsnedders will address open comments and land the PR
*   Finalize the schedule for Interop 2024
    *   Consensus on kicking off the call for proposals on Sept. 14th and to wrap up the proposal selection process by Nov 30th.
*   Discuss voting procedures for the 2024 focus areas [Add a proposal for the Interop 2024 process #390](https://github.com/web-platform-tests/interop/pull/390)
    *   nanrinandu: can we split the PR into 2 so as to unblock the call for proposals
    *   jensimmons: agree that it might be reasonable to split the PR. Last year we did a lot of work and the new proposal is aimed at reducing the effort. We need to discuss how we approach 2023 focus areas.
    *   boazsender: I think there is broad consensus on the proposal, but we have not discussed what to do with Interop 2023 focus areas
    *   chrishtr: assuming carryover is the reasonable approach for 2023 focus areas
    *   jensimmons: we need to come up with a process for how we evaluate what we carryover. Do we retire by consensus or keep by consensus?
    *   nt1m: retiring by consensus makes sense
    *   boazsender: keeping by consensus would be my preference
    *   tantek: I would support jgraham’s position that proposals/focus areas should reapply
    *   Next step: setup some dedicated time to finalize carryover process for 2023 focus areas
*   Communications plan - call for proposals
    *   Review and add comments
